hw/platform/snowy_family/snowy_common.c

```diff
@@ -188,27 +188,27 @@ static void  MemMang_Handler()
 
 void HardFault_Handler()
 {
-    printf("*** HARD FAULT ***\n");
+    KERN_LOG("snowy_common", APP_LOG_LEVEL_ERROR, "*** HARD FAULT ***");
```

Comment:
There is a debate here on hard/usage fault handlers. We should really assume much of the system is dead, and potentially allow the printf to continue. We are dead afterall